cmd/connector-gen/advertise-routes.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"go4.org/netipx"
+)
+
+func advertiseRoutes(set *netipx.IPSet) {
+	fmt.Println()
```

Comment:
I'm assuming this is here to provide some space before the output, but just flagging incase its a mistaken inclusion